model/aws.py

```diff
@@ -28,5 +28,25 @@ def access_key_id(self):
     def secret_access_key(self):
         return self.raw['secret_access_key']
 
+    def rotation_cfg_reference(self) -> model.base.CfgElementReference:
```

Comment:
naming: while the typename is perfectly fine (CfgElementReference), the method's name is not ideal. it is about the cfg-element for rotation. the chosen name, however, drops the `element` part of the name (which reduces emphasis), and puts an emphesis on the aspect of the returned object being a a reference.

please rather name this:
- rotation_cfg_element
- rotation_cfg_element_reference
- rotation_cfg # this would be in sync with other namings of cfg-element-references we already have elsewhere

the same holds true for the attribute-name